src/main/java/au/com/agic/apptesting/steps/FrameStepDefinition.java

```diff
@@ -0,0 +1,79 @@
+package au.com.agic.apptesting.steps;
+
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+
+import au.com.agic.apptesting.State;
+import au.com.agic.apptesting.utils.GetBy;
+import au.com.agic.apptesting.utils.SleepUtils;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import cucumber.api.java.en.When;
+
+/**
+ * This class contains Gherkin step definitions for working with frames/iframes.
+ */
+@Component
+public class FrameStepDefinition {
+
+	@Autowired
+	private SleepUtils sleepUtils;
+
+	@Autowired
+	private GetBy getBy;
+
+	@When("I switch to an iframe indexed at \"(\\d+)\"$")
```

Comment:
no docs?

Reply:
actually, do we need this functionality?

Reply:
how are the frame sets created in the first place? if it's not possible at this time then we don't need to switch between them - this feature can be added then (ie. remove this step definition if we can't make frame sets)